### ISSUE TITLE
Won’t pair with Garmin Venu 3

### ISSUE THREAD

**Issue by 987664987** [intelligenate/dfc#15]
Just received my device and was able to connect through wahoo app as a test but my garmin Venu3 doesn’t find it when searching for a sensor

**Comment by planzi** [comment]
Got mine to work. You have to install the beta firmware on the DTC.

**Comment by 987664987** [comment]
Yes the beta speed/cadence update works but no power reading which supposedly the Venu 3 supports

**Comment by Trebuin** [comment]
There currently isn’t a speed/power/cadence beta available yet.  I would love to have that or at least speed & power to try.  I’m thinking of seeing if it is possible to get two devices & have one pipe out speed & cadence & the second pipe out power & cadence.  Obviously that’s a more expensive choice but not as bad as buying power pedals.  Any news on Speed & Power?  I’m on the Fenix 7 which works great for everything & wouldn’t mind trying betas.

**Comment by mwerner85** [comment]
Confirming some of these comments - I can get the beta Speed/Cadence firmware load to work with my Venu 3, but the base firmware and the beta Power/Cadence one doesn't seem to be recognized by the watch.
